package.json

```diff
@@ -604,6 +734,18 @@
                 "when": "editorTextFocus && neovim.init && neovim.mode != insert || neovim.recording",
                 "args": "<C-Left>"
             },
+            {
+                "command": "vscode-neovim.send",
+                "key": "home",
+                "when": "editorTextFocus && neovim.init && neovim.mode != insert || neovim.recording",
+                "args": "<Home>"
```

Comment:
Would this break native scroll navigation when pressing Home/End?

Reply:
Not sure what you mean. In vscode, home/end moves to the beginning and end of the line - this is what neovim also does.